sw/device/tests/coverage_test.c

```diff
@@ -0,0 +1,43 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/test_coverage.h"
+#include "sw/device/lib/testing/test_main.h"
+#include "sw/device/lib/uart.h"
+#include <stdbool.h>
+#include <stdint.h>
```

Comment:
Standard headers should go before project headers.

Reply:
I think we need to update `.clang-format` for this, but I'm more used to local to global ordering to catch hidden dependencies in headers.

Reply:
I think just putting them in separate stanzas will make clang-format keep way from them. (Style guide actually mandates self-header, system-headers, project-headers, at any rate).

Reply:
> I think just putting them in separate stanzas will make clang-format keep way from them.

Unfortunately it doesn't.

Reply:
Oh weird. I can certainly make system headers come before project headers in the clang format config.

Reply:
I wonder if this is due to the version of the `clang-format` that I have. I can disable it and make this change manually before pushing my latest changes.

Reply:
If CI is passing, don't worry about reformatting. I've got a different file with fixes to the include order in clang-format.

Reply:
Sounds good, making this change manually makes CI unhappy as expected. I will `clang-format` and update this part.